### PR TITLE
Restyle the generator tab

### PR DESCRIPTION
**Pull request by Gara64**
* Use Cozy styleguide for colors
* Use Cozy icons
* Restyle the generator tab
  * Hide all generations options for the user
  * Place the regenerate and copy actions next to the password
* The generated password is automatically copied to the clipboard 
* The password history is more accessible

### CODE REVIEW

File: src/popup/scss/pages.scss [hunk 1]
```diff
@@ -10,8 +10,14 @@ app-sync {
...
     font-size: $font-size-large;
-    font-family: $font-family-monospace;
+    font-family: $font-family-sans-serif;
     margin: 20px;
+    box-sizing: border-box;
+    border-radius: .188rem;
+    border-top: .063rem solid $gray;
+    border-bottom: .063rem solid $gray;
+    border-left: .063rem solid $gray;
+    border-right: .063rem solid $gray;
```
Comment (y-lohse) [inline src/popup/scss/pages.scss]: Any reason not to use `border: .063rem solid $gray;`?

File: src/popup/scss/misc.scss [hunk 2]
```diff
@@ -214,6 +214,31 @@ app-vault-icon {
         background-size: 284px 43px;
     }
 }
+.icon-cozy {
+    display: inline-block;
+    background-size: 16px 16px;
```
Comment (y-lohse) [inline src/popup/scss/misc.scss]: Nit: it's probably possible to use `background-size: contain;` and avoid hardcoding the icon sizes.

File: src/popup/generator/password-generator.component.ts [hunk 3]
```diff
@@ -32,6 +38,8 @@ export class PasswordGeneratorComponent extends BasePasswordGeneratorComponent {
             this.cipherState = addEditCipherInfo.cipher;
         }
         this.showSelect = this.cipherState != null;
+        // Copy the password in the clipboard
+        super.copy();
```
Comment (y-lohse) [inline src/popup/generator/password-generator.component.ts]: Why not `this.copy`?

Comment (Gara64) [reply src/popup/generator/password-generator.component.ts]: I thought it makes more sense to use `super` to call a parent method, but I'm ok with `this` as well if you think it's better

File: src/popup/scss/pages.scss (hunk 1 above)
Comment (Gara64) [reply src/popup/scss/pages.scss]: No real reason, I'll change this

File: src/popup/generator/password-generator.component.ts (hunk 3 above)
Comment (y-lohse) [reply src/popup/generator/password-generator.component.ts]: Yeah I think using `this` is a more conventional way to do it. `super.method` is useful when `method` is redefined in the child class and you really want to access the parent version, but that's pretty rare.

**Comment by Gara64** [comment]
@y-lohse We do not remove options, we hide them between an 'advanced option' button, see the screenshots.
About the automatic copy feature, it is a request from the product, to save the user a click when he wants to generate a new password.

Capture d’écran du 2019-11-13 16-09-10
Capture d’écran du 2019-11-13 16-09-41

**Pull request merged**